aws-ts-organizations/README.md

```diff
@@ -0,0 +1,40 @@
+# AWS Organizations
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new?template=https://github.com/pulumi/examples/blob/master/aws-ts-organizations/README.md)
+
+## Prerequisites
+
+1. [Install Pulumi](https://www.pulumi.com/docs/get-started/install/)
+1. [Configure Pulumi for AWS](https://www.pulumi.com/docs/intro/cloud-providers/aws/setup/)
+
+
+**Note**: This app requires credentials that have permissions to
+AWS Organizations service. The IAM user running this app should
+also be granted permissions to assume the role identified by `OrganizationalAccountAccessRole` in any account.
+
+## Deploying and running the program
+
+Note that unlike other resources that can be created/destroyed easily,
+this app creates an AWS account. Read the blog post about [organizing AWS accounts with Pulumi](https://www.pulumi.com/blog/organizing-aws-accounts-with-pulumi) to learn more.
```

Comment:
This is a tentative link assuming that the blog post slug doesn't change during its review process.